drivers/telescope/lx200ap_v2.cpp

```diff
@@ -1222,6 +874,7 @@ bool LX200AstroPhysicsV2::ReadScopeStatus()
     syncSideOfPier();
 
     // Hack -- when the ra/dec doen't change, the hour angle info isn't being updated. Not sure how to fix...
+    // This may be unnecessary now with changed to KStars. Hy 3/11/2022.
     EqN[AXIS_RA].value = currentRA;
     EqN[AXIS_DE].value = currentDEC;
     IDSetNumber(&EqNP, nullptr);
```

Comment:
Please remove this and use newRADec(...)

Reply:
Done. newRaDec() was already there.